RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -125,4 +141,16 @@ internal class CampaignDispatcher: CampaignDispatcherType, TaskSchedulable {
         return campaignData.intervalBetweenDisplaysInMS ??
             Constants.CampaignMessage.defaultIntervalBetweenDisplaysInMS
     }
+
+    private func data(from url: URL, completion: @escaping (Data?) -> Void) {
+        let sessionConfig = URLSessionConfiguration.default
+        sessionConfig.timeoutIntervalForRequest = 5 // seconds
```

Comment:
I think 5 is reasonable. Let's just put that into a class-level constant (or some `Constants` struct).